Classes/Issues/Comments/IssueCommentSectionController.swift

```diff
@@ -10,15 +10,26 @@ import UIKit
 import IGListKit
 import TUSafariActivity
 import GitHubAPI
+import MessageViewController
```

Comment:
I don't think `IssueCommentSectionController` should know anything about MessageViewController

Reply:
That’s correct! Forgot to remove it before pushing 😄
Thanks!!